.github/workflows/opensk_build.yml

```diff
@@ -24,8 +24,16 @@ jobs:
       - name: Set up OpenSK
         run: ./setup.sh
 
+      - name: Building sha256sum tool
+        uses: actions-rs/cargo@v1
+        with:
+          command: build
+          args: --manifest-path third_party/tock/tools/sha256sum/Cargo.toml
+
       - name: Building OpenSK
         uses: actions-rs/cargo@v1
         with:
           command: build
           args: --release --target=thumbv7em-none-eabi --features with_ctap1
+      - name: Compute SHA-256 sum
+        run: ./third_party/tock/tools/sha256sum/target/debug/sha256sum target/thumbv7em-none-eabi/release/ctap2
```

Comment:
Don't we generate new crypto materials at each build?

Reply:
Good point! We'd need to account for that. Computing the checksum now can help track reproducibility progress though.

Reply:
We could also dump the key materials to the standard output. That way we can reproduce a build and check the hash.

Reply:
We can add some example crypto material as a stub, and use that in the workflow where we want a reproducible build.

Reply:
IIRC there's a way on Github workflows to flag files and save them as produced artefact. Normally this is used to create release packages. Maybe we can experiment around that.

Reply:
Let's do it in a separate pull-request, as it will require more experimentation.